src/Core/Addon/Theme/ThemeManager.php

```diff
@@ -78,7 +139,28 @@ public function upgrade(Addon $theme, $version, $source = null)
      */
     public function enable($name)
     {
-        return true;
+        if (!$this->employee->canEdit()) {
```

Comment:
Should be:

```php
if (!$this->employee->canEdit("AdminThemes", $id_shop)) {
 // stuff that needs authorization
}
```

Or something, but you need to check **what** the employee is allowed to edit.